go/workspace.rst

```diff
@@ -0,0 +1,208 @@
+|                                                                                                  |
+| See http_archive_ for more details.                                                              |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`sha256`            | :type:`string`              | :value:`""`                           |
++----------------------------+-----------------------------+---------------------------------------+
+| The expected SHA-256 hash of the file downloaded.                                                |
+|                                                                                                  |
+| Only valid if :param:`urls` is set.                                                              |
+|                                                                                                  |
+| See http_archive_ for more details.                                                              |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`build_file_name`   | :type:`string`              | :value:`"BUILD.bazel,BUILD"`          |
++----------------------------+-----------------------------+---------------------------------------+
+| The name to use for the generated build files. Defaults to :value:`"BUILD.bazel"`.               |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`                   | :type:`string`              | :value:`"auto"`                       |
+| build_file_generation`     |                             |                                       |
++----------------------------+-----------------------------+---------------------------------------+
+| Used to force build file generation.                                                             |
+|                                                                                                  |
+| * :value:`"off"` : do not generate build files.                                                  |
+| * :value:`"on"` : always run gazelle, even if build files are already present.                   |
+| * :value:`"auto"` : run gazelle only if there is no root build file.                             |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`build_tags`        | :type:`string_list`         | :value:``                             |
++----------------------------+-----------------------------+---------------------------------------+
+| The set of tags to pass to gazelle when generating build files.                                  |
++----------------------------+-----------------------------+---------------------------------------+
+
+go_prefix
+~~~~~~~~~
+
+Set the :param:`importpath` attribute on all rules instead of using `go_prefix`.
```

Comment:
Ditto to `go_prefix` comment above. This should be with regular rules, not workspace rules.

FYI, GitHub doesn't seem to recognize `#721` as an issue link as it does for markdown.